support-frontend/.eslintrc

```diff
@@ -35,6 +35,7 @@
     "import/no-duplicates": "off",
     "import/no-extraneous-dependencies": "off",
     "padded-blocks": "off",
+    "import/first": "off",
```

Comment:
Turn off those annoying `Absolute imports should come before relative imports` errors

Reply:
💯